function-web/src/main/java/com/future/function/web/configuration/ExecutorConfiguration.java

```diff
@@ -0,0 +1,16 @@
+package com.future.function.web.configuration;
+
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class ExecutorConfiguration {
+
+  @Bean
+  public ExecutorService executorService() {
+    return Executors.newFixedThreadPool(5);
```

Comment:
Maybe use properties instead to update the thread pool without redeploy app?

Also, new line before